linux/tests/reals_linux/real_interlocked_renames.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#define interlocked_add                        real_interlocked_add
+#define interlocked_add_64                     real_interlocked_add_64
+#define interlocked_and                        real_interlocked_and
+#define interlocked_and_16                     real_interlocked_and_16
+#define interlocked_and_64                     real_interlocked_and_64
+#define interlocked_and_8                      real_interlocked_and_8
+#define interlocked_compare_exchange           real_interlocked_compare_exchange
+#ifdef _WIN64                          
+#define interlocked_compare_exchange_128       real_interlocked_compare_exchange_128
+#endif                                 

```

Comment:
should also remove

Reply:
removed

---
In reply to: [460143178](https://github.com/Azure/azure-c-pal/pull/16#discussion_r460143178) [](ancestors = 460143178)